### PR TITLE
fully remove captcha from master branch

### LINKED ISSUE #108: Analyze button
Hi,

I tried to deploy the code using Bluemix account and the analyze button does not work. 

https://we-retail-fashion.eu-gb.mybluemix.net/

Any idea ?

Regards,

nishant

### PR DESCRIPTION
**Pull request by kognate**
This fully removes captcha from master.
Fixes #108 in master

### CODE REVIEW

**Comment by nfriedly** [comment]
Should this PR also nix https://github.com/watson-developer-cloud/personality-insights-nodejs/blob/a5dc028d6241f9c8a328886998263c87269620b6/config/security.js#L51 and https://github.com/watson-developer-cloud/personality-insights-nodejs/blob/a5dc028d6241f9c8a328886998263c87269620b6/router/routes/homepage.js#L41 ?

**Comment by kognate** [comment]
Yes, it should. I've removed those lines.

File: config/security.js [hunk 1]
```diff
@@ -48,12 +48,6 @@ module.exports = function (app) {
     next();
   });
 
-  // 5. captcha
-  var captchaKeys = {
-    site: process.env.CAPTCHA_SITE || '<captcha-site>',
-    secret: process.env.CAPTCHA_SECRET || '<captcha-secret>',
-  };
-
   var checkCaptcha = function(req, res, next) {
```
Comment (kognate) [inline config/security.js]: Again you are correct.  I've removed it.

File: config/security.js [hunk 2]
```diff
@@ -48,38 +48,5 @@ module.exports = function (app) {
...
-            error: 'Wrong captcha'
-          });
-        }
-      });
-    } else {
-      next();
-    }
-  };
-
   app.use('/api/', csrfProtection, checkCaptcha, limiter);
```
Comment (nfriedly) [inline config/security.js]: You'll want to kill the checkCaptcha here too. Although I guess I was wrong that you can't remove all of the rest of the file.

(We should also nix CSRF, but that doesn't need to be part of this.)

Comment (kognate) [inline config/security.js]: I'll remove csrf after this one.

**Comment by nfriedly** [comment]
This looks good to me. I haven't tested it locally yet, but there's a lot on my plate today, so I'd say go ahead and merge if it works for you locally.

**Comment by nfriedly** [comment]
Actually, curiosity got the better of me, so I went and spun up a local copy. Everything looks good, merging.

**Pull request merged**